src/main/java/org/commonjava/service/promote/client/kafka/KafkaEventDispatcher.java

```diff
@@ -55,8 +50,7 @@ public void fireEvent( PathsPromoteCompleteEvent event )
         logger.debug( "Firing event to external: {}", event );
         try
         {
-            emitter.send(objectMapper.writeValueAsString(event))
-                    .toCompletableFuture().get( DEFAULT_SYNC_EVENT_TIMEOUT, TimeUnit.MINUTES );
+            emitter.send(objectMapper.writeValueAsString(event));
```

Comment:
Do we need to convert it to `String` or send the object itself ?

Reply:
I send it as String for safe. I did not add any serialization methods.

Reply:
For emitter, no need to custom the serialization and just config it as `io.quarkus.kafka.client.serialization.ObjectMapperSerializer` in application.yaml. And sure we can test it first.

Reply:
I guess the config do the same job. Yeah, let's test this first.